tests/saitests/py3/sai_qos_tests.py

```diff
@@ -4514,7 +4514,7 @@ def runTest(self):
                 send_packet(self, src_port_id, pkt, pkt_num)
 
                 # Account for leakout
-                if 'cisco-8000' in asic_type:
+                if 'cisco-8000' in asic_type and not is_multi_asic:
```

Comment:
can you check why their is no impact for pizza box T0/T1 ?

Reply:
below change is needed for single asic to make it more accurate.

Reply:
The code doesn't change for the T0/T1. The PR changes code only for mulitasic - if it is single asic, the logic is still same.